governance/paper-process.md

```diff
@@ -0,0 +1,235 @@
+# Process for creating papers
+
+This document is intended to provide a consistent mechanism for cloud native
+environmental sustainability to produce community papers, ensure they are reviewed, and subsequently
+published.  Content of this document does not supersede existing processes and
+is intended to be used in conjunction with [existing proposal to project
+instructions](process.md).
+
+## Proposal
+
+If a proposal is made that includes a paper as a deliverable, the proposal needs
+to ensure that there is a clearly identified lead and a well defined paper
+scope.
+
+The paper scope and topic should be raised in at least one WG meeting to
+solicit more volunteers (ideally 4).  Interested parties should meet at least
+once to describe the intent of the paper, & propose a very rough outline to
+present to WG leadership for planning and scheduling as a project.
+
+## Project
+
+Once a WG Leadership sponsor is assigned to the project, the group should meet
+to and agree on a tentative schedule.
+
+### Tentative schedule milestones
+
+The below list the minimum milestones that should be captured in a tentative
+schedule along with the estimated time frame for completion.  Milestones are
+explained further in this document.
+
+| TODO | Milestone | Estimated time |
+| -- | -- | -- |
+| * [ ] | Audience, Goals, & refining scope | 1 week |
+| * [ ] | Tasking Assignment | 1 week |
+| * [ ] | Content Rough-in | 2-3 weeks |
```

Comment:
I don't think `Rough-in` offers meaningful clarity in this context. Did you mean "outline" or "draft"?